cra-template/template.json

```diff
@@ -1,28 +1,29 @@
 {
-    "package": {
-      "dependencies": {
-        "@luigi-project/client": "^1.25.1",
-        "@luigi-project/core": "^1.25.1",
-        "react": "^18.2.0",
-        "react-dom": "^18.2.0",
-        "react-router-dom": "^6.4.3"
-      },
-      "devDependencies": {
-        "@babel/core": "^7.20.2",
-        "@babel/preset-env": "^7.20.2",
-        "@babel/preset-react": "^7.18.6",
-        "babel-loader": "^9.1.0",
-        "copy-webpack-plugin": "^11.0.0",
-        "html-webpack-plugin": "^5.5.0",
-        "webpack": "^5.75.0",
-        "webpack-cli": "^5.0.0",
-        "webpack-dev-server": "^4.11.0"
-      },
-      "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "start": "webpack-dev-server --mode development",
-        "build": "webpack --mode production",
-        "serve": "webpack-dev-server --mode production"
-      }
+  "package": {
+    "dependencies": {
+      "@luigi-project/client": "^2.13.0",
+      "@luigi-project/core": "^2.13.0",
+      "react": "^18.3.1",
+      "react-dom": "^18.3.1",
+      "react-router-dom": "^6.23.1",
+      "react-scripts": "5.0.1"
```

Comment:
If this react-scripts dependency was not needed before, I would remove it. Unless removing it brakes something. We dont use them in the scripts section it seems